src/main/resources/db/changelog/data/financial_transactions.csv

```diff
@@ -1,9 +1,12 @@
-id,wallet_id,transaction_type,amount,transaction_date,description
-1,1,INCOME,15000,2022-02-19T09:47:52.595721658,""
-2,2,EXPENSE,5000,2022-02-19T09:47:52.595721658,null
-3,2,EXPENSE,5000,2022-02-19T09:47:52.595721658,description
-4,2,EXPENSE,5000,2022-02-19T09:47:52.595721658,description
-5,3,INCOME,5000,2022-02-19T09:47:52.595721658,description
-6,3,INCOME,4000,2022-02-19T09:47:52.595721658,description
-7,4,INCOME,15000,2022-02-19T09:47:52.595721658,description
-8,4,INCOME,2000,2022-02-19T09:47:52.595721658,description
+id,wallet_id,transaction_type,amount,transaction_date,description,financial_transaction_category_id
+1,1,INCOME,15000,2022-02-19T09:47:52.595721658,"",""
```

Comment:
Field financial_transaction_category_id is number, not string

Reply:
done